app/src/lib/app-state.ts

```diff
@@ -134,6 +134,9 @@ export interface IAppState {
 
   /** Whether we should show a confirmation dialog */
   readonly confirmRepoRemoval: boolean
+
+  /** The external editor to use when opening repositories */
```

Comment:
📖 Could we make this clearer that it's the _name_ of the external editor and not, for example, the bundle ID or registry key?